stats/Record.md

```diff
@@ -44,26 +44,34 @@ measure name instead of the `Measure`.
 
 Implementations MAY define a `MeasurementMap` which describes a set of data points to be collected
 for a set of Measures. Adding this functionality may improve the efficiency of the record usage API.
-Additionally, when recording Measurements, `MeasurementMap` should optionally take a map of string 
-key-value pairs to record an exemplar. The string map is called `attachments` and represents the 
+Additionally, when recording Measurements, `MeasurementMap` should optionally take a map of string
+key-value pairs to record an exemplar. The string map is called `attachments` and represents the
 contextual information of an exemplar, for example trace id, span id or dropped labels.
 
 ## Recording Stats
 
-Users should record Measurements against a context, either an explicit context or the implicit 
-current context. Tags from the context are recorded with the Measurements if they are any. 
+Users should record Measurements against a context, either an explicit context or the implicit
+current context. Tags from the context are recorded with the Measurements if they are any.
 When recording against an explicit context, implementations should allow users to add extra tags,
 and those tags should not be added to the current context.
 
 Note that there is no implicit recording for exemplars. If you want to record a `Measurement`
 against an exemplar, you have to explicitly pass a string-string map.
 
-Implementations SHOULD provide a means of recording multiple Measurements at once. This 
+Implementations SHOULD provide a means of recording multiple Measurements at once. This
 functionality can be provided through one of the following options:
 * As a method in a class/package (recommended name Stats/stats), taking a list of Measurement as
 argument. e.g. `record(List<Measurement>)` or `record(...Measurement)`.
 * As a `record` method of the appropriate data type. e.g. `MeasurementMap.record()`.
 
+### Record semantics
+
+Record may defer view updates to a background thread or process. Therefore, updates to views
+may not take effect right away. No guarantees are necessarily provided about the order in
+which Record calls are processed.
+
+In particular, in case of overload (e.g. views cannot be updated fast enough), Record may drop data.
```

Comment:
Lets not have this sentence in the current PR and if needed open an issue to discuss sampling/dropping records policies.